exercises/gigasecond/instructions.md

```diff
@@ -0,0 +1,5 @@
+# Instructions
+
+Your task is to determine the date and time one gigasecond after a certain date.
+
+For example, if you were born on _January 24th, 2015 at 22:00 (10:00:00pm)_, then you would be a gigasecond old on _October 2nd, 2046 at 23:46:40 (11:46:40pm)_.
```

Comment:
It would be cool if, for people who don't want to do too much reading, all of the information necessary to solve the exercise was in instructions. Here, the definition of a gigasecond is missing.

Reply:
Yepp, that's a good point. I'll add that as a guiding principle.